change/office-ui-fabric-react-2020-02-27-19-36-04-GroupedListCustomCheckbox.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "patch",
```

Comment:
This would have to be `minor` since we're introducing API

Reply:
Thank you, I fixed this!